docs/userguide/optimizing.rst

```diff
@@ -48,22 +48,6 @@ like adding new worker nodes, or revoking unnecessary tasks.
 General Settings
 ================
 
-.. _optimizing-librabbitmq:
-
-librabbitmq
------------
-
-If you're using RabbitMQ (AMQP) as the broker then you can install the
-:pypi:`librabbitmq` module to use an optimized client written in C:
-
-.. code-block:: console
-
-    $ pip install librabbitmq
-
-The 'amqp' transport will automatically use the librabbitmq module if it's
```

Comment:
This part still seems to be true, see kombu's code: https://github.com/celery/kombu/blob/81101877a09e8f4442555c6c74b4836f113e4f51/kombu/connection.py#L251-L252

#4693 says that librabbitmq is no longer the recommended package, but the code in kombu hasn't be updated to stop automatically converting the `amqp://` prefix to `librabbitmq://` if it is installed.

Reply:
we need to update kombu as well

Reply:
I've made a [PR on kombu](https://github.com/celery/kombu/pull/1225) to address this.